src/Model/Process.php

```diff
@@ -292,7 +292,17 @@ public static function stop($kill = false, $id = null)
 	 * @return bool Returns true if the process is inactive
 	 */
 	public static function checkInactive($id) {
-		$entry = dba::selectFirst('process', ['pid'], ['id' => $id, 'parent' => self::$instance->parent]);
+		if (is_null(self::$instance)) {
+			return false;
```

Comment:
Is this a silent failure? Maybe raise a loud exception here?

Reply:
No, because maybe it's "just" a process of another node. So I return "false" because I cannot know if the process is inactive